src/assets/mock-data/hep.json

```diff
@@ -1,6 +1,7 @@
 {
   "$schema": "http://json-schema.org/schema#",
   "description": "An article or thesis or book or...",
+  "x_editor_validation_url": "./assets/mock-data/remote-validation-response.json",
```

Comment:
As we mentioned IRL this should be stored in a JSON that should be passed to the editor and overlayed on top of the schema.